docs/mine/lotus/dagstore.md

```diff
@@ -43,18 +43,18 @@ Here's a definition of terms:
 - **Shard state:** the state that a dagstore shard is in. It can be one of the  following:
   - `ShardStateNew`: indicates that a shard has been registered, but has not yet been initialized.
   - `ShardStateInitializing`: indicates that the shard is being initialized.
-    - `ShardStateAvailable`: indicates that the shard has been initialized and is capable of serving retrievals. However, there are no active shard readers in this instant.
+    - `ShardStateAvailable`: indicates that the shard has been initialized and is capable of serving retrievals. However, there are no active shard readers in this instance.
```

Comment:
The previous meaning was correct, but just in case I'll use a synonym:

```suggestion
    - `ShardStateAvailable`: indicates that the shard has been initialized and is capable of serving retrievals. However, there are no active shard readers in this moment.
```